docs/api.rst

```diff
@@ -9073,7 +9073,7 @@ Adding blockchain accounts
 
       This endpoint can also be queried asynchronously by using ``"async_query": true``
 
-   Doing a PUT on the the blockchains endpoint with a specific blockchain URL and a list of account data in the json data will add these accounts to the tracked accounts for the given blockchain and the current user. The updated balances after the account additions are returned.
+   Doing a PUT on the blockchains endpoint with a specific blockchain URL and a list of account data in the json data will add these accounts to the tracked accounts for the given blockchain and the current user. A list of accounts that are in rotki is returned.
```

Comment:
Actually, none of these :) I updated the docs and the returned data is the list of accounts' addresses that were added during a request. (this list is returned in case caller added an ens name and needs it's resolved address for the further usage).